src/plugins/kibana/public/dashboard/components/panel/panel.js

```diff
@@ -52,6 +52,14 @@ define(function (require) {
             var uiState = panelConfig.uiState || {};
             $scope.uiState = $scope.parentUiState.createChild(getPanelId(panelConfig.panel), uiState, true);
 
+            if ($scope.uiState) {
+              $scope.panel.customTitle = $scope.uiState.get('title');
+              // sync external uiState changes
+              var syncUIState = () => $scope.panel.customTitle = $scope.uiState.get('title');
```

Comment:
Don't use arrow syntax in this file. We're slowly moving things to ES6, but this file has not been moved